clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -45,10 +48,12 @@
 public class ConsumerNetworkClient implements Closeable {
     private static final Logger log = LoggerFactory.getLogger(ConsumerNetworkClient.class);
 
+    private final ReentrantLock lock = new ReentrantLock();
```

Comment:
Same comment as with other class -- would be good to identify what is being protected here since we have a lock, a concurrent queue, and not everything is locked...